### PR TITLE
Open source friends page

### PR DESCRIPTION
**Pull request by AmoabaKelvin**
Completes #263

### CODE REVIEW

File: apps/web/src/config/ossFriends.ts
Comment (mxkaske) [inline apps/web/src/config/ossFriends.ts:1]: To stick to our file naming convention, let's go for `oss-friends.ts`

File: apps/web/src/app/oss-friends/page.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                </CardTitle>
+              </div>
+              <CardDescription>{friend.description}</CardDescription>
+            </CardHeader>
+            <CardFooter>
+              <div className="flex items-center gap-2.5">
+                <Link href={friend.website} target="_blank">
+                  <Icons.globe className="text-muted-foreground h-6 w-6 hover:text-black" />
+                </Link>
+                {friend.socialLinks.twitter && (
```
Comment (mxkaske) [inline apps/web/src/app/oss-friends/page.tsx]: I wonder if we can do

```
Object.entries(friends.socialLinks).map(([key, value]) => <Link key={key} {...} />)
```

To future proof more links

**Review by mxkaske — COMMENTED** [review]
Small remarks. Great work! 🔥

File: apps/web/src/app/oss-friends/page.tsx (hunk 1 above)
Comment (AmoabaKelvin) [reply apps/web/src/app/oss-friends/page.tsx]: Updated them all @mxkaske

File: apps/web/src/config/oss-friends.ts
Comment (mxkaske) [inline apps/web/src/config/oss-friends.ts:1]: We can type safe it a bit more by moving `TwitterIcon` to the `Icons` and use the `ValidIcon` property.

```ts
import { Icons, ValidIcon } from "@/components/icons";

type SocialLink = {
  icon: ValidIcon;
  link: string;
};
```

You will only need to update the `icon` to use the object key
```
twitter: {
  icon: "twitter",
  link: "https://twitter.com/documenso",
},
```

and inside of the `CardFooter` use

```ts
const Icon = Icons[value.icon];
```

**Review by mxkaske — COMMENTED** [review]
Last comment to remove `any` and maybe we can reduce the size of the icons a bit. Other than that lgtm!

**Pull request merged**